optimum/exporters/onnx/config.py

```diff
@@ -211,3 +212,88 @@ def generate_dummy_inputs_for_validation(self, reference_model_inputs: Mapping[s
             reference_model_inputs["encoder_hidden_states"] = reference_model_inputs.pop("encoder_outputs")[0]
 
         return reference_model_inputs
+
+
+class EncoderDecoderOnnxConfig(OnnxSeq2SeqConfigWithPast):
+    DUMMY_INPUT_GENERATOR_CLASSES = (
+        DummyTextInputGenerator,
+        DummySeq2SeqDecoderTextInputGenerator,
+    )
+
+    def __init__(
+        self,
+        config: "PretrainedConfig",
+        task: str = "default",
+        patching_specs: Optional[List["PatchingSpec"]] = None,
+        use_past: bool = False,
+        use_past_in_inputs: Optional[bool] = None,
+        use_present_in_outputs: Optional[bool] = None,
+        behavior: ConfigBehavior = ConfigBehavior.MONOLITH,
+    ):
+        super().__init__(
+            config,
+            task=task,
+            patching_specs=patching_specs,
+            use_past=use_past,
+            use_past_in_inputs=use_past_in_inputs,
+            use_present_in_outputs=use_present_in_outputs,
+            behavior=behavior,
+        )
+
+        from ..tasks import TasksManager
```

Comment:
would we have it at the top level?

Reply:
It causes circular export on top

Reply:
This could mean that some refactor is needed, I guess.

Reply:
Deferring import would still be ok IMO. But if there is a better way would look into it.